clients/banking/src/components/RecurringTransferList.tsx

```diff
@@ -741,20 +716,16 @@ export const RecurringTransferList = ({
               onPress={reload}
             />
 
-            <Space width={8} />
-          </Box>
-
-          <Space height={12} />
+            <Fill />
```

Comment:
nit: put a `minWidth` prop to add a small space when the two items are near (ex: on mobile)